# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../pg_filedump/make_patch.diff
# Copyright (C) 2007 - 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./contrib/Makefile.orig	2012-02-23 23:53:36.000000000 +0100
+++ ./contrib/Makefile	2012-05-09 15:44:39.655157960 +0200
@@ -37,6 +37,7 @@
 		pg_test_fsync	\
 		pg_trgm		\
 		pg_upgrade	\
+		pg_filedump-9.1.0 \
 		pg_upgrade_support \
 		pgbench		\
 		pgcrypto	\

